src/style/blocks/card.css

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-img {
```

Comment:
(it was mentor's comment on my commit) - we don't need to use "product" part in classes naming

as example: not "card__product-img", but "card__img"